src/main/kotlin/no/nav/familie/ef/sak/behandling/grunnbelop/GOmregningTaskService.kt

```diff
@@ -4,23 +4,30 @@ import no.nav.familie.ef.sak.beregning.nyesteGrunnbeløpGyldigFraOgMed
 import no.nav.familie.ef.sak.fagsak.FagsakRepository
 import no.nav.familie.ef.sak.infrastruktur.exception.feilHvisIkke
 import no.nav.familie.ef.sak.infrastruktur.featuretoggle.FeatureToggleService
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.scheduling.annotation.Scheduled
 import org.springframework.stereotype.Service
 
 @Service
 class GOmregningTaskService(val fagsakRepository: FagsakRepository,
                             val featureToggleService: FeatureToggleService,
                             val gOmregningTask: GOmregningTask) {
 
+    val logger: Logger = LoggerFactory.getLogger(this::class.java)
+
+    @Scheduled(cron = "0 0 15 23 5 ?")
```

Comment:
Er det greit hvis vi trigger denne manuellt i stedet? 
Vet ikke om vi trenger at den skal kjøre hvert år den 23 mai? 
2 podder som starter denne, og det kommer feile for en av de
```suggestion
```

Reply:
Hvis vi skal trigge manuelt, mp vi ha en bruker i saksbehandling som har tilgant til å kalle endepunktet, eller legge ut endepunktet unprotected i prod. Tenkte også det kunne være greit å sette opp jobben til å kjøre regelmessig for å plukke opp skaer som blir lukket etter hvert.

Reply:
Høres slitsomt ut hvis vi må deploye en ny versjon hver gang vi skal trigge jobben? 
Unprotected har vi brukt tidligere for liknende saker og det eneste den gjør er ju å opprette tasks

Reply:
`Unprotected` har vi brukt tidligere for liknende saker og det eneste den gjør er ju å opprette tasks.
Det går å ha den som `protected` sånn at vi kan featuretoggle hvem som skal kunne bruke den å la Mirja trigge denne.

Hvis ikke må vi deploye en ny versjon hver gang vi skal trigge den, og hvis det skulle være nedetid akkurat då, eller feiler av noen grunn, så må vi trigge den på nytt igjen